ios/magnet/AppDelegate.m

```diff
@@ -52,8 +55,19 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
   UIViewController *rootViewController = [UIViewController new];
   rootViewController.view = rootView;
   self.window.rootViewController = rootViewController;
+  [NotificationsHelper register];
   [self.window makeKeyAndVisible];
   return YES;
 }
 
+- (void)applicationDidBecomeActive:(UIApplication *)application {
+  [NotificationsHelper clearNotifications];
+  [NotificationsHelper disable];
+}
+
+- (void)applicationDidEnterBackground:(UIApplication *)application {
+  [NotificationsHelper clearNotifications];
```

Comment:
Trust me, with timers in the middle, better to be sure, specially cause what we are executing is not really resource consuming.